src/services/webclient.ts

```diff
@@ -1227,6 +1228,24 @@ export class WebClientService {
     }
 
     /**
+     * Remove all messages of a receiver
+     * @param {threema.Receiver} receiver
+     * @returns {Promise<any>}
+     */
+    public clean(receiver: threema.Receiver): Promise<any> {
+        if (receiver === undefined) {
```

Comment:
We should probably check for `null` too. (The joys of Javascript.)

Reply:
feel free to append this check :laughing: